build.gradle

```diff
@@ -76,6 +75,7 @@ task integration(type: Test) {
   testClassesDirs = sourceSets.integrationTest.output.classesDirs
   classpath = sourceSets.integrationTest.runtimeClasspath
   failFast = true
+  testLogging.showStandardStreams = false
```

Comment:
this can be removed now

Reply:
Prefer to have it there as it's easy to turn flag on locally to see the logs, rather adding the whole property when needed.